src/topology/algebra/continuous_functions.lean

```diff
@@ -125,13 +244,15 @@ In this section we show that continuous functions valued in a topological algebr
 obtained by requiring that `A` be both a `topological_semimodule` and a `topological_semiring`
 (by now we require `topological_ring`: see TODO below).-/
 
+section subtype
+
```

Comment:
Can we just remove this section now? Or is it still needed elsewhere?

Can we at least mark it as deprecated?

Reply:
Yes, I'd mark it as deprecated for a couple of reasons. First I did not substitute `continuous_map` to the subtype of continuous functions everywhere in the library because I was not sure this is what people wanted: I need to use continuous bundled maps but maybe someone needs to use the subtype. For example, `continuous_map` does not allow you to state that continuous functions valued in a topological group are a subgroup of all functions, so something is lost. To me it is fine if the two implementations leave parallel and when there is no specific reason to use the subtype, for the sake of uniformity, people use the bundled functions. This is what I say in the first paragraph of the documentation of this file. If you believe the subtype should not exist anymore, I'd merge this PR and then you or I can do another PR to replace the subtype everywhere in the library and then remove it.